RPS_ver2.py

```diff
@@ -1,10 +1,16 @@
+import sys
```

Comment:
We're only using this so we can exit gracefully (no more ctrl+c to kill process or whatever it has been you're doing)